src/rpcgovernance.cpp

```diff
@@ -654,3 +654,23 @@ UniValue getsuperblockbudget(const UniValue& params, bool fHelp)
 
     return strBudget;
 }
+
+UniValue getsuperblockheight(const UniValue& params, bool fHelp)
+{
+    if (fHelp) {
+        throw runtime_error(
+            "getsuperblockheight\n"
```

Comment:
I like the idea of having such rpc however I think it probably should be more generic rather than work for active chain tip only. How about asking for a optional blockheight here (and having default as a current height)?